### PR TITLE
Update go-bitfield

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

**Review by magik6k — APPROVED** [review]
(wait for tests)

File: go.mod [hunk 1]
```diff
@@ -23,7 +23,7 @@ require (
 	github.com/fatih/color v1.8.0
 	github.com/filecoin-project/filecoin-ffi v0.30.4-0.20200716204036-cddc56607e1d
 	github.com/filecoin-project/go-address v0.0.4
-	github.com/filecoin-project/go-bitfield v0.2.1-0.20200920172649-837cbe6a1ed3
+	github.com/filecoin-project/go-bitfield v0.2.0
```
Comment (Stebalien) [inline go.mod:26]: @Kubuxu note: this _doesn't_ include the buffer pool change, in case you still wanted that.

Comment (Kubuxu) [reply go.mod:26]: ahh, I used latest instead of an explicit tag, and go proxy didn't pick it up yet

**Review by magik6k — APPROVED** [review]
Looks more better

**Comment by Kubuxu** [comment]
I love that `test-short` takes longer than `test`. Probably will fail there but it is unrelated. It is the same bitfield code after all.

**Pull request merged**